bindings/dotnetcore/dotnet-core-binding/Microsoft.Azure.Devices.Gateway/Gateway.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Azure.Devices.Gateway
+{
+    /// <summary>
+    ///  Wrapper for Azure IoT Gateway SDK. 
+    /// </summary>
+    public class Gateway
```

Comment:
We have Broker Interop today which is very similar to this code. So, I'm calling this Gateway Interop.